src/foam/u2/dialog/StyledModal.js

```diff
@@ -146,7 +146,7 @@ foam.CLASS({
             .start().addClasses(['h400', this.myClass('title')]).add(this.title).end()
             .start()
               .addClass(this.myClass('modal-body'))
-              .tag('', null, this.content$)
+              .add(this.addBody())
```

Comment:
So you didn't end up  having to change content?

Reply:
No, had a talk with Kevin and he said this was the way to do it. Just changing content was causing DOM loading issues